src/netif.c

```diff
@@ -43,6 +43,9 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <ipvs/redirect.h>
+#ifdef CONFIG_ICMP_FWD_CORE
```

Comment:
I think `CONFIG_ICMP_REDIRECT_CORE` is better, since the local-in ICMP packets are also redirected to the specified worker core.

So is the config name `icmp_redirect_core`  is better than `icmp_fwd_core`.

Reply:
Has been modified, use CONFIG_ICMP_REDIRECT_CORE.